applications/annotate_points.md

```diff
@@ -0,0 +1,314 @@
+            """Keybinding to decrement to the previous label with wraparound"""
+            current_properties = points_layer.current_properties
+            current_label = current_properties['label'][0]
+            ind = list(labels).index(current_label)
+            n_labels = len(labels)
+            new_ind = ((ind - 1) + n_labels) % n_labels
+            new_label = labels[new_ind]
+            current_properties['label'] = np.array([new_label])
+            points_layer.current_properties = current_properties
+        
+```
+## point_annotator()
+We will create the GUI within a function called `point_annotator()`. Wrapping the GUI creation in the function allows us to integrate it into other functions (e.g., a command line interface) and applications. See below for the function definition.
+
+```python
+def point_annotator(
+	im_path: str,
+	labels: List[str],
+):
+    """Create a GUI for annotating points in a series of images.
+
+    Parameters
+    ----------
+    im_path : str
+        glob-like string for the images to be labeled.
+    labels : List[str]
+        list of the labels for each keypoint to be annotated (e.g., the body parts to be labeled).
+    """
+
+```
+
+## Loading the video
+First, we load the movie to be annotated. Since behavior movies can be quite long, we will use a lazy loading strategy (i.e., we will only load the frames as they are used). Using [`dask-image`](https://github.com/dask/dask-image), we can construct an object that we can pass to napari for lazy loading in just one line. For more explanation on using dask to lazily load images in napari, see [this](https://napari.org/tutorials/applications/dask) tutorial.
+
+```python
+stack = imread(im_path)
+```
+
+We can then start the viewer Note that we use the `napari.gui_qt()` context manager to start and manage Qt event loop. We must provide the `contrast_limits` to prevent the image stack from being evaluated (i.e., loaded) upon layer creation. All of the following GUI code should be within the `napari.gui_qt()` context manager.
+
+```python
+with napari.gui_qt():
+    viewer = napari.view_image(stack, contrast_limits=[0, 255])
+```
+
+## Annotating with points
+We will annotate the features of interest using points in a napari Points layer. Each feature will be given a different label so that we can track them across frames. To achieve this, we will store the label in the `Points.properties` property in the 'label' key. We will instantiate the `Points` layer without any points. However, we will initialize `Points.properties` with the property values we will be using to annotate the images. To do so, we will define a properties dictionary with a key named `label` and values `labels`. The key, 'label', is the name of the property we are storing which feature of interest each point corresponds with. The values, 'labels', is the list of the names of the features we will annotating (defined above in the "point_annotator()" section). 
```

Comment:
the `Points.properties` property -> the `Points.properties` attribute?

also, "we will annotating" -> "we will be annotating"

Reply:
Fixed!